packages/cf-deploy-config-writer/src/translations/cf-deploy-config-writer.i18n.json

```
@@ -14,9 +14,12 @@
         "cleanupModules": "Cleanup modules.",
         "creatingMta": "Creating MTA using {{- cdsParams }}...",
         "html5AppAdded": "HTML5 application added to MTA configuration {{- appName}}",
+        "html5AlreadyExists": "HTML5 application {{- appName}} already exists in the MTA configuration.",
         "addXsuaaService": "Adding XSUAA service to MTA configuration.",
         "mtaReadWithPrefix": "Read mta.yaml with prefix {{- prefix }}.",
-        "errorReadingMta": "Error reading mta.yaml file {{- error }}."
+        "errorReadingMta": "Error reading mta.yaml file {{- error }}.",
+        "unableToReadPackageJson": "Unable to read package.json file {{- error }}.",
```

```suggestion
        "unableToReadPackageJson": "Unable to read the `package.json` file {{- error }}.",
```
What should users do to try to fix this error? View the logs?

---

There is nothing we can do! Normally its related to incorrectly formatted JSON which is rare but this would fail the build process much earlier anyway. A 2nd reason possibly is related to a missing package.json, again really ad-hoc to have a project without a package.json.